PreFreeSurfer/scripts/AnatomicalAverage.sh

```diff
@@ -36,6 +36,8 @@ opts_AddOptional '--working-dir' 'wdir' 'dir' "temporary working directory"
 
```

Comment:
This script has been changed to non-executable by the PR, please `chmod +x` it and commit again.

Reply:
Several other scripts in this PR have also been changed to non-executable.